browser/components/CodeEditor.js

```diff
@@ -683,6 +708,24 @@ export default class CodeEditor extends React.Component {
     })
   }
 
+  initialHighlighting () {
+    if (this.editor.options.linesHighlighted == null) {
+      return
+    }
+
+    const count = this.editor.lineCount()
+    for (let i = 0; i < count; i++) {
```

Comment:
Will it be faster if we just loop the highlighted lines?
```js
const totalHighlightedLines = this.editor.options.linesHighlighted.length;
const totalAvailableLines = this.editor.lineCount();
for (let i = 0; i < totalHighlightedLines; i++){
  const lineNumber = this.editor.options.linesHighlighted[i];
  if (lineNumber > totalAvailableLines) {
    // make sure that we skip the invalid lines althrough this case should not be happened.
   continue;
  }
  this.editor.addLineClass(lineNumber, 'text', 'CodeMirror-activeline-background');
}
```

Reply:
Thank you for the fast response and review.
Yes, your changes make it better optimized.
I will make the changes to the code.